sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -93,13 +134,7 @@ internal class RadarApiClient(
 
         apiHelper.request(context, "GET", url, headers, null, false, object : RadarApiHelper.RadarApiCallback {
             override fun onComplete(status: RadarStatus, res: JSONObject?) {
-                if (res != null && res.has("meta")) {
-                    val meta = res.getJSONObject("meta")
-                    if (meta.has("config")) {
-                        val config = meta.getJSONObject("config")
-                        RadarSettings.setConfig(context, config)
-                    }
-                }
+                callback?.onComplete(status, res, parseMeta(res))
```

Comment:
Seems like it might be good to leave the code that sets the config in settings. Could this new callback object return a config object? Or if the callback is null, could this code still execute (then instead of defaulting the callback to null, you'd default it to an anonymous inner class that calls this existing code, perhaps)?

Reply:
`config` is not used at the moment (nor is it sent down by the server). I also felt it was appropriate to pass these through the callback as it doesn't feel like the responsibility of the `RadarApiClient` to perform this side-effect